CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Current Master
 
+- Remove BlacklistController from disk state
```

Comment:
Update to include recent blocks, or just generalize “removed unneeded data from storage”